### PR TITLE
Header component implemented

### PR DESCRIPTION
**Pull request by Randula98**
image
image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@headlessui/react": "^1.6.4",
+    "@headlessui/vue": "^1.6.4",
```
Comment (ThusharaX) [inline package.json:7]: Please remove necessary dependencies

**Review by ThusharaX — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  },
+  {
+    name: 'Rules',
+    href: '#',
+  },
+  {
+    name: 'Contact',
+    href: '#',
+  },
+]
```
Comment (rusiruavb) [inline src/components/Header/Header.jsx:36]: Missing semicolon

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import { Fragment } from 'react'
+import { Popover, Transition } from '@headlessui/react'
+import {
+  MenuIcon,
+  XIcon,
+} from '@heroicons/react/outline'
+
```
Comment (rusiruavb) [inline src/components/Header/Header.jsx:7]: Please add semicolons at the end of each expression or install **Prettier** extension to VS Code. That will automatically add the semicolons for you

File: src/index.js [hunk 4]
```diff
@@ -3,9 +3,12 @@ import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
 
+import Header from "./components/Header/Header";
+
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
 	<React.StrictMode>
+		<Header />
```
Comment (rusiruavb) [inline src/index.js:11]: Do not add the Header component here. Add it to the App component.

**Review by rusiruavb — CHANGES_REQUESTED** [review]
@Randula98 use `yarn` to install the dependencies. `package-lock.json` should not create if you use **yarn**. Remove the package-lock.json and use yarn to install packages.

**Comment by Randula98** [comment]
@rusiruavb noted ayya. Thank you..

**Pull request merged**